### PR TITLE
WDC crash course

### PR DESCRIPTION
**Pull request by Samuel-Baird**
I'm new to using Github, so please give this a good look to make sure I formatted everything correctly!

Thanks @nsilvestri for giving me an overview of how to make this :)

### CODE REVIEW

File: edudoc/disciplinary-crash-course
Comment (nsilvestri) [inline edudoc/disciplinary-crash-course:1]: This file needs to be removed, since it seems to be an accident.

File: documents/motions/15.2022.1 - Suspensions and other Sanctions.md [hunk 1]
```diff
@@ -20,7 +20,7 @@
    7. to reinstate a Registered Speedcuber who has been sanctioned under 1.1 to 1.6, before the end of the fixed period or before the set of circumstances specified has changed or ceased to exist. WDC must communicate each reinstatement to the WCA Board with a clear motivation.
 2. The WDC may exercise its suspensive rights under this Motion only if:
    1. a Registered Speedcuber has failed to pay fees for WCA Competitions, or
-   2. in the opinion of the WDC, the Registered Speedcuber is in breach of the Bylaws, Motions, or Regulations, or
+   2. in the opinion of the WDC, the Registered Speedcuber is in breach of the Bylaws, Motions, Regulations, or Code of Conduct, or
```
Comment (nsilvestri) [inline documents/motions/15.2022.1 - Suspensions and other Sanctions.md:23]: This change seems irrelevant, and should be removed from this PR.

File: edudoc/disciplinary-crash-course.md
Comment (nsilvestri) [inline edudoc/disciplinary-crash-course.md:1]: There should be another directory on this file path, and the file name converted to snake_case: `edudoc/disciplinary-crash-course/disciplinary_crash_course.md`

File: edudoc/disciplinary-crash-course.md [hunk 2]
```diff
@@ -0,0 +1,96 @@
+## WCA Disciplinary Crash Course
```
Comment (nsilvestri) [inline edudoc/disciplinary-crash-course.md:1]: The title of the document should have only one `#`, to indicate the highest level section title.

File: edudoc/disciplinary-crash-course.md [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+
+# Disciplinary
```
Comment (nsilvestri) [inline edudoc/disciplinary-crash-course.md:3]: Since this isn't the title of the document, this should have two hashes: `## Disciplinary`.

On another note, I'm not sure that "Disciplinary" is a great title for this section or document? It's not really descriptive about what this section is about. If it's just an intro, you can get away with not having a section either.

File: edudoc/disciplinary-crash-course.md [hunk 4]
```diff
@@ -0,0 +1,96 @@
...
+
+The WCA Disciplinary Committee (WDC) is entitled to apply bans and other disciplinary action as a result of violation of the [WCA Code of Conduct](https://www.worldcubeassociation.org/documents/Code%20of%20Conduct.pdf), based on the motion [Suspensions and other Sanctions](https://www.worldcubeassociation.org/documents/motions/15.2022.1%20-%20Suspensions%20and%20other%20Sanctions.pdf). However, the Committee is not able to handle difficult situations quickly during a competition, especially if communication with multiple involved parties is required. Cooperation with competition Delegates is often necessary. This document [or chapter, or section, or whatever this is going to be] contains a collection of guidelines to instruct new Delegates on how to handle common disciplinary incidents, what disciplinary measures they can take and what they should report to the WDC.
```
Comment (nsilvestri) [inline edudoc/disciplinary-crash-course.md:5]: > [or chapter, or section, or whatever this is going to be]

Since it has been determined what this is going to be, this parenthetical should be removed.

> ...on how to handle common disciplinary incidents, what disciplinary measures they can take and what they should report to the WDC.

nit: I like Oxford commas.

File: edudoc/disciplinary-crash-course.md [hunk 5]
```diff
@@ -0,0 +1,96 @@
...
+
+# Disciplinary Log and Banned Competitors
```
Comment (nsilvestri) [inline edudoc/disciplinary-crash-course.md:7]: \##

File: edudoc/disciplinary-crash-course.md [hunk 6]
```diff
@@ -0,0 +1,96 @@
...
+
+* A list of banned competitors can be found in the [Disciplinary log}(https://www.worldcubeassociation.org/disciplinary). If a competitor attempts to attend a competition when they are banned from doing so, please contact the WDC.
```
Comment (nsilvestri) [inline edudoc/disciplinary-crash-course.md:9]: The `}` should be a `]`.

Also, how should a delegate contact WDC? If they are a delegate, they don't (currently) have access to Slack. The Unofficial WCA Staff Discord is currently unofficial, and until it gets some other level of approval from Board+, it's probably best to not officially endorse it.

File: edudoc/disciplinary-crash-course.md [hunk 7]
```diff
@@ -0,0 +1,96 @@
...
+* Depending on the degree of severity and type of violation of the Code of Conduct, the WDC may ban a competitor from competing, or attending competitions. Such a ban will be communicated with local Delegates when it is issued. If a competitor is banned from competing, but not from attending competitions, they are permitted to attend competitions. They may also fulfill the roles of officials at competitions at the discretion of the WCA Delegate.
```
Comment (nsilvestri) [inline edudoc/disciplinary-crash-course.md:10]: > the WDC may ban a competitor from competing, or attending competitions.

Can remove the comma.

> They may also fulfill the roles of officials at competitions

Presumably we aren't talking about a banned Delegate performing delegate duties at a competition here, so it may be better to use the word "staff" here.

File: edudoc/disciplinary-crash-course.md [hunk 8]
```diff
@@ -0,0 +1,96 @@
...
+* The WDC will post reports of particularly noteworthy cases that they have handled, as well as a quarterly digest to highlight smaller cases resolved in the last quarter. These are posted for the purpose of educating the community and WCA staff, as well as transparency. These will be posted in the announcement section on the WCA homepage and the WCA forum; a complete list of these posts and announcements can be found in the [Disciplinary log](https://www.worldcubeassociation.org/disciplinary).
```
Comment (nsilvestri) [inline edudoc/disciplinary-crash-course.md:11]: I understand what this paragraph is saying but I think it could be written a bit more clearly:

> The WDC will publish reports on remarkable cases they've managed and issue a quarterly digest showcasing smaller resolved cases from the previous quarter. These publications aim to educate the community and WCA staff, promoting transparency. You can find them in the announcement section on the WCA homepage and forum, with a comprehensive list available in the Disciplinary log.

File: edudoc/disciplinary-crash-course.md [hunk 9]
```diff
@@ -0,0 +1,96 @@
...
+
+# Handling Behavioral Issues of Competitors at Competitions 
+
+* Delegates should primarily address minor behavioral issues of competitors at competitions themselves using [2j](https://www.worldcubeassociation.org/regulations/#2j), [2k](https://www.worldcubeassociation.org/regulations/#2k), and [2l](https://www.worldcubeassociation.org/regulations/#2l) of the regulations and applicable sub-points.
+  * If Delegates have not first tried to handle minor behavior issues by giving warnings and disqualifying competitors under regulation 2k, then the WDC is much less likely to apply any disciplinary action. We would like Delegates to attempt to handle minor incidents at a local level first before WDC gets involved. However, the WDC would like Delegates to report noteworthy behavioral issues so the WDC can record them. The WDC is always willing to give advice, and take disciplinary action as needed.
```
Comment (nsilvestri) [inline edudoc/disciplinary-crash-course.md:16]: > The WDC is always willing to give advice, and take disciplinary action as needed.

Don't need the comma.

File: edudoc/disciplinary-crash-course.md [hunk 10]
```diff
@@ -0,0 +1,96 @@
...
+* If any extraordinary cases occur, please describe them in the Delegate report.
```
Comment (nsilvestri) [inline edudoc/disciplinary-crash-course.md:17]: Is there a set of guidelines for what constitutes an extraordinary case?

File: edudoc/disciplinary-crash-course.md [hunk 11]
```diff
@@ -0,0 +1,96 @@
...
+  * Please be aware the WDC will likely contact you for further information regarding the case.
+  * If an incident is discovered after the competition, this should be added to the Delegate report, or emailed to the Delegate report thread.
+
+  # {.page-break-before}
```
Comment (nsilvestri) [inline edudoc/disciplinary-crash-course.md:21]: In general we should only use these tags when starting new sections.

File: edudoc/disciplinary-crash-course.md [hunk 12]
```diff
@@ -0,0 +1,96 @@
...
+
+* Delegates should primarily address minor behavioral issues of competitors at competitions themselves using [2j](https://www.worldcubeassociation.org/regulations/#2j), [2k](https://www.worldcubeassociation.org/regulations/#2k), and [2l](https://www.worldcubeassociation.org/regulations/#2l) of the regulations and applicable sub-points.
+  * If Delegates have not first tried to handle minor behavior issues by giving warnings and disqualifying competitors under regulation 2k, then the WDC is much less likely to apply any disciplinary action. We would like Delegates to attempt to handle minor incidents at a local level first before WDC gets involved. However, the WDC would like Delegates to report noteworthy behavioral issues so the WDC can record them. The WDC is always willing to give advice, and take disciplinary action as needed.
+* If any extraordinary cases occur, please describe them in the Delegate report.
+  * Please be aware the WDC will likely contact you for further information regarding the case.
+  * If an incident is discovered after the competition, this should be added to the Delegate report, or emailed to the Delegate report thread.
```
Comment (nsilvestri) [inline edudoc/disciplinary-crash-course.md:19]: > Delegate report thread

Capitalization here is a bit jarring. There's no style guide for how to refer to delegates, but I'd either capitalize both words like (Delegate Report) or none (delegate report).

File: edudoc/disciplinary-crash-course.md [hunk 13]
```diff
@@ -0,0 +1,96 @@
...
+
+  # {.page-break-before}
+
+  * If the case contains information of a sensitive or private nature, it is acceptable to exclude these details from the Delegate report. However, it is important that the WDC is contacted directly with these details.
```
Comment (nsilvestri) [inline edudoc/disciplinary-crash-course.md:23]: Delegate Report capitalization

File: edudoc/disciplinary-crash-course.md [hunk 14]
```diff
@@ -0,0 +1,96 @@
...
+* Please contact the WDC (tagging in report or directly):
+  * If the conflict cannot be resolved locally.
+  * If you believe that a person should be officially warned by the WDC or banned from competitions.
```
Comment (nsilvestri) [inline edudoc/disciplinary-crash-course.md:26]: When should a delegate decide that a person needs to be officially warned?

File: edudoc/disciplinary-crash-course.md [hunk 15]
```diff
@@ -0,0 +1,96 @@
...
+  * If you believe that similar incidents may occur in the future and that it would be beneficial for the WDC to document the incident.
+* For extreme situations where competitor safety may be involved, please contact local law enforcement immediately. Once the immediate concerns of safety have been addressed, contact the WDC with all evidence gathered for the WDC to use in their investigation.
+
+# Theft at Competitions
```
Comment (nsilvestri) [inline edudoc/disciplinary-crash-course.md:30]: \##

File: edudoc/disciplinary-crash-course.md [hunk 16]
```diff
@@ -0,0 +1,96 @@
...
+
+* The WDC’s preference is for theft (such as stealing puzzles) to be investigated and resolved locally when possible, as the local Delegates can often resolve these cases better than the WDC. The priority for these cases is getting items returned to owners during the competition. The WDC will generally not apply disciplinary action unless a competitor has been caught stealing after a previous warning or all of the following conditions are met:
+  * There is definitive proof of the accused person committing theft (such as a witness seeing them steal, CCTV footage, admission of guilt, etc.).
+  * The suspect has been contacted and asked to return stolen items.
+  * The suspect has been unresponsive/uncooperative and has not returned stolen items.
+* If you receive reports that lead you to believe that there has been theft, then do your best to resolve the situation during the competition if possible.
+  * Make an announcement, giving a description of stolen items and advising competitors to give items to an organizer or Delegate if they are found. In past incidents, competitors have returned stolen items after similar announcements.
```
Comment (nsilvestri) [inline edudoc/disciplinary-crash-course.md:37]: Until there is proof of theft, perhaps it would be better to refer to the puzzles as "missing" rather than "stolen"? Many people mix-up cubes or grab things that aren't theirs by accident, which is distinct from theft.

File: edudoc/disciplinary-crash-course.md [hunk 17]
```diff
@@ -0,0 +1,96 @@
...
+
+# Handling Behavioral Issues of Spectators at Competitions {.page-break-before}
+
+* Local Delegates should do their best to handle any issues that arise with spectators in a respectful manner, remembering that they are acting as official representatives of the WCA.
+* Be aware of venue rules and policies. You may be able to ask an official from the venue or local law enforcement to remove individuals causing problems if needed. 
+* Contact local law enforcement immediately if there is extreme behavior that may threaten competitor safety. Once the immediate concerns of safety have been addressed, contact the WDC to report the details of what happened.
+
+# Online Interactions (WCA Forum/Official WCA Platforms)
+
+* If competitors/parents/others are being impolite or inappropriate in their online interactions related to competitions, then please do your best to respectfully handle the situation (please keep in mind that you are a representative of the WCA). You may reach out to the WDC for advice on how to best handle the situation if the problem persists or you are unsure of how to proceed.
```
Comment (nsilvestri) [inline edudoc/disciplinary-crash-course.md:52]: I think it is worth mentioning that the WDC's jurisdiction specifically does not extend outside of WCA-controlled environments before this point is made, as it is not clear up to this point when reading the document for the first time.

File: edudoc/disciplinary-crash-course.md [hunk 18]
```diff
@@ -0,0 +1,96 @@
...
+* If you feel a person may engage in disruptive actions or pose a threat to the safety or well-being or anyone during a WCA-sanctioned competition, then please contact the WDC.
```
Comment (nsilvestri) [inline edudoc/disciplinary-crash-course.md:53]: This seems out of place in the section titled "Online Interactions", since it specifically lists "during a WCA-sanctioned competition"

File: edudoc/disciplinary-crash-course.md [hunk 19]
```diff
@@ -0,0 +1,96 @@
...
+
+# Organizers
+
+* Like any other competitors, report any violations of the Code of Conduct to the WDC.
+* If an organizer has done something outside of the scope of the Code of Conduct that you feel could be problematic, then it is up to discretion of the Delegate to work with that organizer or not. For example, if an organizer has made racist comments on a non-WCA online forum, then the Delegate can choose to not work with that organizer. 
+* Carefully keep track of competition budgets and actual expenditures that take place, documenting expenditures with receipts when possible. If you believe that an organizer or legal entity is retaining leftover money from competitions for personal gain then please contact the WDC.
+
+# WCA Staff Members {.page-break-before}
+
+* Like any other competitors, report any violations of the Code of Conduct of WCA Staff to the WDC. Be sure to also use ‘CC’ to include the WCA Ethics Committee (WEC) in your email.
```
Comment (nsilvestri) [inline edudoc/disciplinary-crash-course.md:64]: It is worth explicitly making the comparison to situations where one would report a staff member to WEC instead of WDC, even if it's technically out of scope of this document. Right now it seems to imply that all staff infractions should go to WDC, unless they are on WDC themselves.

File: edudoc/disciplinary-crash-course.md [hunk 20]
```diff
@@ -0,0 +1,96 @@
...
+* Tag the WDC in the Delegate report about the case. Give a rough outline of the incident for the education of other Delegates, but avoid including any evidence or identifying information in the report such as videos that reveal the identity of competitors or names of competitors accused of cheating.
+* Contact the WDC directly with a detailed description of the incident, and any evidence gathered.
+
+# Incidents Outside the Scope of the Code of Conduct
+The Code of Conduct is limited in scope to WCA Competitions and online on official WCA platforms. As a result, the WDC cannot take any action in response to incidents that happen outside of this scope. If concerning behavior happens outside of this scope, below are some recommendations.
+* Contact authority figures that may be able to resolve the situation, such as parents, school administrators, or moderators of a social media group.
+# {.page-break-before}
```
Comment (nsilvestri) [inline edudoc/disciplinary-crash-course.md:82]: Remove this page break

File: edudoc/disciplinary-crash-course.md [hunk 21]
```diff
@@ -0,0 +1,96 @@
...
+* Tag the WDC in the Delegate report about the case. Give a rough outline of the incident for the education of other Delegates, but avoid including any evidence or identifying information in the report such as videos that reveal the identity of competitors or names of competitors accused of cheating.
+* Contact the WDC directly with a detailed description of the incident, and any evidence gathered.
+
+# Incidents Outside the Scope of the Code of Conduct
+The Code of Conduct is limited in scope to WCA Competitions and online on official WCA platforms. As a result, the WDC cannot take any action in response to incidents that happen outside of this scope. If concerning behavior happens outside of this scope, below are some recommendations.
+* Contact authority figures that may be able to resolve the situation, such as parents, school administrators, or moderators of a social media group.
```
Comment (nsilvestri) [inline edudoc/disciplinary-crash-course.md:81]: There needs to be a new line before a new set of bullet points, or else they just get added on to the end of the previous sentence. Annoying markdown stuff.

File: edudoc/disciplinary-crash-course.md [hunk 22]
```diff
@@ -0,0 +1,96 @@
...
+# {.page-break-before}
+
+* If the situation is serious enough (e.g. physical threats, bullying, abusive behavior, sexual harassment) then contact local law enforcement.
+* If you feel it is necessary, communicate with the individual about their behavior and contact the WDC, keeping in mind that the WDC may not be able to apply sanctions. In your communications make it clear that you are contacting them as a concerned individual and not a representative of the WCA. For example, do not threaten disciplinary action from the WCA, such as removal from competitions.
+  * If harassment/bullying is occurring online, advise the competitor to block the perpetrator(s).
```
Comment (nsilvestri) [inline edudoc/disciplinary-crash-course.md:86]: The victim may not always be a competitor, so "competitor" may be better phrased as "target", or something similar.

File: edudoc/disciplinary-crash-course.md [hunk 23]
```diff
@@ -0,0 +1,96 @@
...
+
+# Managing Difficult Situations
+Competitions can be demanding, difficult situations that can add another level of stress. It is essential to remain professional at all times; this will help your interactions with others to go more smoothly, and allow you to positively represent the WCA. Keep in mind the following points when managing a difficult situation:
+* Remain calm during your interactions, speak slowly, avoid retaliating, avoid shouting, don’t use overly complicated jargon, and be empathetic.
```
Comment (nsilvestri) [inline edudoc/disciplinary-crash-course.md:90]: Add a newline above this line.

> avoid retaliating

What does this mean?

> be empathetic

What does this look like in such an interaction?

In general, this bullet point does a lot of telling the delegate _what_ to do, but not _why_ they would do it. I think it would be more educational explaining why their demeanor is important when handling these situations.

File: edudoc/disciplinary-crash-course.md [hunk 24]
```diff
@@ -0,0 +1,96 @@
...
+* If you are imposing an unpopular ruling, explain clearly why you are doing it, if the competitor is young, it may be beneficial to involve their parent/guardian. Draw from personal experience, if this has happened to you previously, tell them.
```
Comment (nsilvestri) [inline edudoc/disciplinary-crash-course.md:91]:
```suggestion
* If you are imposing an unpopular ruling, explain clearly why you are doing it. If the competitor is young, it may be beneficial to involve their parent/guardian. Some competitors are easily embarrassed. One way to defuse that tension is to draw from personal experience and let them know that the same situation has happened to you before.
```

On the other hand, is this totally relevant to WDC? It's not really a disciplinary issue, and would fit better in the Handbook.

**Review by nsilvestri — CHANGES_REQUESTED** [review]
I didn't note all formatting issues on titles, but they generally all need to be adjusted to have two hashes, except when otherwise noted.

Another note on style: this document uses "The WDC" instead of just "WDC". I general hear WCA committees and teams referred to by just their acronym, without the definitive article.

File: edudoc/disciplinary-crash-course.md
Comment (Samuel-Baird) [reply edudoc/disciplinary-crash-course.md:1]: I'm not exactly sure what this means :P I will need some clarification

Comment (nsilvestri) [reply edudoc/disciplinary-crash-course.md:1]: You fixed it in the other PR. In essence I was asking you to create a folder for the document like the others one have. Resolved!

File: edudoc/disciplinary-crash-course.md (hunk 7 above)
Comment (Samuel-Baird) [reply edudoc/disciplinary-crash-course.md:10]: Since “staff” isn’t a defined term, we have opted to go with the term “officials” since that is what the regulations use.

also, if a delegate gets banned then they are pretty likely to get demoted from their delegate status so that shouldn’t ever be an issue :)

File: edudoc/disciplinary-crash-course.md (hunk 10 above)
Comment (Samuel-Baird) [reply edudoc/disciplinary-crash-course.md:17]: We didn’t define that but hoped that this section would help delegates realize that some behavior is minor and shouldn’t really need WDC involvement, while other behavior should need it. We considered some examples but decided that we could publish as is and look into creating an exhaustive list down the road. Once you get to the disciplinary section of the delegate handbook it would be good to reach out to us about this :)

File: edudoc/disciplinary-crash-course.md (hunk 12 above)
Comment (Samuel-Baird) [reply edudoc/disciplinary-crash-course.md:19]: Other documents and the regulations use double capitals so I will adjust this to reflect that :)

File: edudoc/disciplinary-crash-course.md (hunk 14 above)
Comment (Samuel-Baird) [reply edudoc/disciplinary-crash-course.md:26]: It’s just supposed to cover anything not covered by the other points. It’s very vague, but we are trying to help delegates realize that the purpose of contacting us is generally to apply disciplinary action or to put someone’s name on our “watch list”

File: edudoc/disciplinary-crash-course.md (hunk 18 above)
Comment (Samuel-Baird) [reply edudoc/disciplinary-crash-course.md:53]: Added “based on their online behavior” to clarify this

File: edudoc/disciplinary-crash-course.md (hunk 19 above)
Comment (Samuel-Baird) [reply edudoc/disciplinary-crash-course.md:64]: That is correct, if a staff member violates CoC then it should go to the WDC. The WDC will forward it to the WEC if it’s an ethical issue as well

File: edudoc/disciplinary-crash-course.md (hunk 11 above)
Comment (Samuel-Baird) [reply edudoc/disciplinary-crash-course.md:21]: If I don’t do this then I believe the rest of the text won’t fit on this page, is there a way to address that?

File: edudoc/disciplinary-crash-course.md (hunk 6 above)
Comment (Samuel-Baird) [reply edudoc/disciplinary-crash-course.md:9]: We are leaving it intentionally open since situations where people are banned from attending can be more serious. Anytime communication is mentioned in the WCA it almost always refers to email so the delegates should already be aware of that option, but if they have a faster form of communication like the discord server, the contact info of a WDC member, etc. then they are welcome to use those methods as well

File: edudoc/disciplinary-crash-course.md (hunk 3 above)
Comment (Samuel-Baird) [reply edudoc/disciplinary-crash-course.md:3]: We are still working on some names, that was mostly a place holder :p

I was thinking something to the effect of "Guidelines for handling behavioral issues and cheating".

File: edudoc/disciplinary-crash-course.md (hunk 6 above)
Comment (nsilvestri) [reply edudoc/disciplinary-crash-course.md:9]: Good point. With the Discord and Slack I kinda forgot email existed haha.

File: edudoc/disciplinary-crash-course.md (hunk 11 above)
Comment (nsilvestri) [reply edudoc/disciplinary-crash-course.md:21]: I guess I'm ok with this in order to avoid orphan bullet points bleeding across page breaks.
